apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -695,4 +708,15 @@ public Mono<List<WorkspaceAndResourceId>> getTraceWorkspace(
                 .collectList();
     }
 
+    @Override
+    @com.newrelic.api.agent.Trace(dispatcher = true)
+    public Flux<TraceCountResponse.WorkspaceTracesCount> countTracesPerWorkspace(Connection connection) {
+
+        var statement = connection.createStatement(TRACE_COUNT_BY_WORKSPACE_ID);
+
+        return Mono.from(statement.execute())
+                .flatMapMany(result -> result.map((row, rowMetadata) -> new TraceCountResponse.WorkspaceTracesCount(
+                        row.get("workspace_id", String.class),
+                        row.get("traces_count", Integer.class))));
```

Comment:
Minor: you probably want to `Long` instead of `Integer`.

Reply:
Integer is enough here I guess, considering that we count traces per day.